### DISCUSSION TITLE
黑鷹直升機無人化再進一步，首度成功執行救援任務

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#936] [General]
美國陸軍、DARPA 與賽考斯基公司今年合作進行黑鷹直升機無人化測試，繼先前完成基本飛行和運貨任務測試後，近日更進一步完成緊急救援示範任務，朝實用範圍邁進。
這次測試無人機和 AI 戰鬥載具在未來戰爭中的潛力，讓美軍非常滿意，代表了陸軍在未來作戰中，AI 無人系統有潛力擔任許多重要任務。

我認為AI被用在軍事上是無法避免的，我曾經讀過YOLO的論文，原作者在v3版本後就不再推出新版本的原因是因為技術被用在軍事上有點超脫他所期望的範圍，雖然後續民間還是有很多持續再進行研究，但能看的出來作者是不希望自己的研究結果被拿來打仗的，不過我認為只要這世界上有兩個國家以上就無法避免。

https://technews.tw/2022/11/08/ai-blackhawk-succefully-tested-in-rescue-mission/

**Comment by peggy0329** [comment]
這個AI無人系統也太厲害！
但還是希望運用在無人機的用途上可以以救援或是緊急運送物品為主
用在戰爭上的話，不知道未來的世界會變成怎麼樣...
會不會有一場無人機大戰，好可怕

**Comment by StevenWanglolz** [comment]
不知道發明這些科技的人知道它們的發明被用於戰爭會作何感想:( 科技的進步是要造福人類的，不是讓人們互相殘殺啊!

**Comment by Evan0979** [comment]
如果無人直升機救援失敗,那責任該歸屬在誰身上呢?感覺會很有爭議......

**Comment by CPCpyt** [comment]
雖然希望世界和平，不過這樣的技術運用在戰爭中的確可以改變戰爭的規模和形態。
被運用在戰爭中是無法避免的結果，不過在那之外可以盡量開發正向的運用，改善人們的日常。
